apps/meeting/src/providers/VideoTileGridProvider/index.tsx

```diff
@@ -108,7 +108,10 @@ const VideoTileGridProvider: React.FC = ({ children }) => {
     };
 
     priorityBasedPolicy.addObserver(observer);
-
+    dispatch( {
```

Comment:
In FF, the priority policy will be undefined so should we dispatch only if valid here?

Reply:
well its a demo so don't think we need to do validation here. The validation if needed should be included in the library.

Reply:
I haven't tried the demo yet, but will this impact the QA testing?

Reply:
Not sure what you mean, they can toggle only choose this when testing in non-FF browser right? (this is done as requested from QA)

Reply:
oh right, I forget that part. 😝

Reply:
Yeah just want to check if this is to be looked upon if not that important should be fine.